feat: Remove PostgreSQL component (#2561)
codenvy/che-docs: modules/administration-guide/pages/installing-che-using-storage-classes.adoc

* A storage type
* And many others

> {prod-short} has the only component that require persistent volumes to store data:

Reviewer: ```suggestion
{prod-short} has one component that requires persistent volumes to store data:
```

>   * A {prod-short} workspaces. {prod-short} workspaces store source code using volumes, for example `/projects` volume.

Reviewer: ```suggestion
  * A {prod-short} workspace. {prod-short} workspaces store source code using volumes, for example `/projects` volume.
```
